snap/snapcraft.yaml

```diff
@@ -4,8 +4,114 @@ summary: OpenStack Exporter for Prometheus
 license: MIT
 contact: solutions-engineering@lists.canonical.com
 description: |
-  The OpenStack exporter, exports Prometheus metrics from a running OpenStack
-  cloud for consumption by prometheus.
+  This snap uses openstack-exporter to provide OpenStack cloud metrics via Prometheus.
+  Grafana dashboards can then be used to visualize the exported metrics, see for example:\
+  https://grafana.com/grafana/dashboards/9701-openstack-dashboard/
+
+  **How-To**
```

Comment:
Snap store markdown doesn't support headings?

Reply:
Nope, the supported markdown is really lean
![image](https://github.com/user-attachments/assets/822ce160-d93f-408b-a318-ec5c89156d6c)

Reply:
You can check the dcgm's docs and the store page for reference.
https://snapcraft.io/dcgm
https://github.com/canonical/dcgm-snap/tree/snap/snapcraft.yaml

Reply:
It's annoying

Reply:
Ah yep sorry, I did just check that. :+1: